docs/guides/plans-overview.mdx

```diff
@@ -13,7 +13,20 @@ This documentation is relevant to the new IBM Quantum&reg; Platform. If you need
 
 IBM Quantum Platform has Open (free access), and paid plans, as described below. See [IBM Quantum access plans](https://www.ibm.com/quantum/pricing) for full details.
 
-If you want access to multiple plans, you can create multiple instances and assign different plans to them.  For example, you can have an Open Plan instance and a Standard Plan instance.
+If you want access to multiple plans, you can create multiple instances and assign different plans to them.  For example, you can have an Open Plan instance and a Standard Plan instance.  You can set prefered instances to use in the `QiskitRuntimeService` initializer or `QiskitRuntimeService.save_account()` method.  Example:
```

Comment:
```suggestion
If you want access to multiple plans, you can create multiple instances and assign different plans to them.  For example, you can have an Open Plan instance and a Standard Plan instance.  You can set preferred instances to use in the `QiskitRuntimeService` initializer or `QiskitRuntimeService.save_account()` method.  Example:
```

Reply:
@Eric-Arellano another example of spellcheck failing? Will tag this in spellcheck issue